sparkbench/ml/src/main/scala/com/intel/sparkbench/ml/LinearRegressionDataGenerator.scala

```diff
@@ -75,6 +73,7 @@ object LinearRegressionDataGenerator {
   }
 
   def main(args: Array[String]) {
+    println(s"start !!")
```

Comment:
This file should be left unchanged?

Reply:
@gczsjdy  Thanks , I will remove it that debug code.